esp/scm/ws_access.ecm

```diff
@@ -995,6 +1032,7 @@ ESPservice [version("1.10"), auth_feature("NONE"), exceptions_inline("./smc_xslt
     ESPmethod GroupQuery(GroupQueryRequest, GroupQueryResponse);
     ESPmethod GroupMemberQuery(GroupMemberQueryRequest, GroupMemberQueryResponse);
     ESPmethod ResourceQuery(ResourceQueryRequest, ResourceQueryResponse);
+    ESPmethod ResourcePermissionQuery(ResourcePermissionQueryRequest, ResourcePermissionQueryResponse);
```

Comment:
do you need to version protect this with "min_ver" ?

Reply:
Will add...